### ISSUE TITLE
Is the project still alive?

### ISSUE THREAD

**Issue by Stvad** [MoonMail/MoonMail#473]
Hey! I'm considering using the MoonMail, but I see that there were no activity on this repo for ~1.5y now. So I wanted to check in on the state of the project!

**Comment by ebisbe** [comment]
@Stvad same as me. Did you end up using it? I'd like to know

**Comment by Stvad** [comment]
No =/

**Comment by ebisbe** [comment]
Yeah. I reached to the same conclussion by reading a few other issues in this same repository. Are you using any other tool? I'm considering Mautic or Sendy or just search a better mailchimp option

**Comment by Stvad** [comment]
I ended up using Mailgun for that use-case, which is ok, but I'm not sure if it was the most optimal solution 🤷‍♂️

**Comment by llwp** [comment]
I've enjoyed using Mautic fwiw. Very robust. Was looking for a lighter option for a particular use case but this project looks headed to retirement.... tbh

**Comment by ebisbe** [comment]
I might end up using AWS PinPoint

**Comment by MaazAhmadSatti3** [comment]
any update?

**Comment by ebisbe** [comment]
It's dead
